include/common_types.hpp

```diff
@@ -15,6 +15,7 @@ At runtime just dlopen the dll, that's it!
 namespace chainblocks {
 
 enum class BasicTypes {
+  None,
```

Comment:
Is it needed (explicitly) or is it to avoid `Any` having the value `0`?

Reply:
I needed it in `(Substrate.Decode` for `nil` values actually